planning/behavior_path_planner/README.md

```diff
@@ -195,54 +196,153 @@ The Pull Over module is activated when goal is in the shoulder lane. Ego-vehicle
+
+##### **geometric parallel parking**
+
+Generate two arc paths with discontinuous curvature. It stops twice in the middle of the path to control the steer on the spot. There are two path generation methods: forward and backward.
+See also [[1]](https://www.sciencedirect.com/science/article/pii/S1474667015347431) for details of the algorithm. There is also [a simple python implementation](https://github.com/kosuke55/geometric-parallel-parking).
+
+###### Parameters geometric parallel parking
+
+| Name              | Unit  | Type   | Description                             | Default value |
+| :---------------- | :---- | :----- | :-------------------------------------- | :------------ |
+| arc_path_interval | [m]   | double | interval between arc path points        | 1.0           |
+| max_steer_rad     | [rad] | double | maximum steer angle for path generation | 0.35          |
+
+###### arc forward parking
+
+Generate two forward arc paths.
+
+![arc_forward_parking](./image/arc_forward_parking.drawio.svg)
+
+[Video of how arc_forward_parking works](https://user-images.githubusercontent.com/39142679/178034128-4754c401-8aff-4745-b69a-4a69ca29ce4b.mp4)
+
+###### Parameters arc forward parking
+
+| Name                                    | Unit  | Type   | Description                                      | Default value |
+| :-------------------------------------- | :---- | :----- | :----------------------------------------------- | :------------ |
+| enable_arc_forward_parking              | [-]   | bool   | flag whether to enable arc forward parking       | true          |
+| after_forward_parking_straight_distance | [m]   | double | straight line distance after pull over end point | 2.0           |
+| forward_parking_velocity                | [m/s] | double | velocity when forward parking                    | 0.3           |
+
+###### arc backward parking
+
+Generate two backward arc paths.
+
+![arc_backward_parking](./image/arc_backward_parking.drawio.svg).
+
+[Video of how arc_forward_parking works](https://user-images.githubusercontent.com/39142679/178034280-4b6754fe-3981-4aee-b5e0-970f34563c6d.mp4)
+
+###### Parameters arc backward parking
+
+| Name                                     | Unit  | Type   | Description                                      | Default value |
+| :--------------------------------------- | :---- | :----- | :----------------------------------------------- | :------------ |
+| enable_arc_backward_parking              | [-]   | bool   | flag whether to enable arc backward parking      | false         |
+| after_backward_parking_straight_distance | [m]   | double | straight line distance after pull over end point | 2.0           |
+| backward_parking_velocity                | [m/s] | double | velocity when forward parking                    | -0.3          |
```

Comment:
```suggestion
| backward_parking_velocity                | [m/s] | double | velocity when backward parking                   | -0.3          |
```

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/1578/commits/0e7de761312528ac65945ae0e8292ed67ff146a3